Information Sharing/Dashboard Accessibility Guidance.md

```diff
@@ -0,0 +1,161 @@
+  * NOT be redundant or provide the exact same information as text within the context of the image.
+  * NOT use the phrases "image of ..." or "graphic of ..." to describe the image. It’s usually apparent to the user that it is an image. And if the image is conveying content, it is typically not necessary that the user knows that it is an image that is conveying the content, as opposed to text.
+
+## Chart design and layout
+
+* All chart elements should have sufficient colour contrast (see [Contrast (minimum) WCAG 1.4.3](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)).
+  * Make sure your chart works in greyscale.
+  * Blue is a rich colour across types of colour blindness so is a good base colour for charts.
+* If gridlines are to be used in graphs, they should always be grey. A good tone is #bebebe (R190 G190 B190).
+* Dashed lines can be used for graphs but no more than two variations.
+* Avoid pattern fill.
+* Charts should have horizontal text, never vertical. Consider swapping the chart axis if long labels are a problem.
+* Double value axes should not be used. Use two charts instead.
+* Include instructions for interacting with charts.
+* Chart titles should be coded as text, not as images. Include a title above the chart to ensure that this is read by a screen reader prior to the chart.
+* Output resolution should be set to take account of intended destination. If a chart is pasted from Excel into Word and resized, the image can become pixelated and font size can fall below required size.
+* Ensure that chart elements will not be clipped or hidden when users change display options – for example, text spacing, resolution, resizing, contrast (see [Reflow WCAG 1.4.10](https://www.w3.org/WAI/WCAG21/Understanding/reflow.html) for more guidance, particularly around avoiding losing functionality beyond 400% zoom).
+* Do not do the following:
+  * use colour alone to convey information (see [Use of colour WCAG 1.4.1](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html)). Alternative options include:
+    * adding labels
+    * mapping values to shapes
+    * splitting up charts
+  * use subtle colour changes to indicate extreme outliers
+  * rely solely on sensory characteristics of components such as shape, color, size, visual location, orientation, or sound (see [Sensory characteristics WCAG 1.3.3](https://www.w3.org/WAI/WCAG21/Understanding/sensory-characteristics.html))
+  * have internal scroll bars on large chunks of text o use borders and backgrounds for charts
+  * use text that is too small
+  * use white space/images as padding
+  * map tool tips underlinked like links when they are not links.
+
+## Branding and corporate colours
+
+Logo must be presented in accordance with PHS branding guidelines and style guide. The Public Health Scotland logo should be used. Logos of legacy organisations should not be used.
```

Comment:
might be obvious but is it worth just stating that these colours already meet accessibility guidelines (I assume they do anyway!)